packages/swirl-components/src/components/swirl-button/swirl-button.tsx

```diff
@@ -191,6 +192,9 @@ export class SwirlButton {
             </span>
           )}
           {!hideLabel && <span class="button__label">{this.label}</span>}
+          {Boolean(this.tag) && (
```

Comment:
Issue: We moved to prefer slots for these scenarios (see icon slot above). Would be great, if you could replace the prop with a slot, as well.